src/js/view/utils/views-manager.js

```diff
@@ -21,7 +21,7 @@ function lazyInitIntersectionObserver() {
                     for (let j = views.length; j--;) {
                         let view = views[j];
                         if (entry.target === view.getContainer()) {
-                            view.model.set('intersectionRatio', entry.intersectionRatio);
+                            view.setIntersectionRatio(entry.intersectionRatio);
```

Comment:
I would like us to add widgets to the views-manager in a different array. SO we can have:

```js
const views = [];
const widgets = [];
```

and for adding and removing:
```js
 export default {	 export default {
     add: function(view) {
         views.push(view);
     },
     remove: function(view) {
         const index = views.indexOf(view);
         if (index !== -1) {
             views.splice(index, 1);
         }
     },
     addWidget: function(widget) {
         widgets.push(widget);
     },
     removeWidget: function(widget) {
         const index = widgets.indexOf(widget);
         if (index !== -1) {
             widgets.splice(index, 1);
         }
     },
```